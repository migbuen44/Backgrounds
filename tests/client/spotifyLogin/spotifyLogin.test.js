import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import SpotifyLogin from '../../../client/src/components/spotifyLogin/spotifyLogin';

describe('SpotifyLogin', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SpotifyLogin />, div)
  })

  it('matches snapshot', () => {
    const tree = renderer.create(<SpotifyLogin />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
