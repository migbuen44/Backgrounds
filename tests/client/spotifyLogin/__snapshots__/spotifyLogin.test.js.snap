// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotifyLogin matches snapshot 1`] = `
<a
  className="lib-spotifyLogin-module-spotifyLoginButton-2601"
  href="https://accounts.spotify.com/authorize?client_id=4cb458aef9d344d2a58c62e7da3d0da5&response_type=code&redirect_uri=https://backgroundsmb.com/&scope=streaming%20user-read-email%20user-read-private%20user-library-read%20user-library-modify%20user-read-playback-state%20user-modify-playback-state"
>
  Spotify Login
</a>
`;
